src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Dispute.java

```diff
@@ -322,9 +322,9 @@ public class Dispute extends Auditable<String> {
 	 * The disputant requires spoken language interpreter. The language name is
 	 * indicated in this field.
 	 */
-	@Column
+	@Column(length = 3)
```

Comment:
Should we also add swagger open api spec validation and Oracle API bean validation like this since we know the exact max length:
@Size(max = 3)
@Column(length = 3)
@Schema(maxLength = 3, nullable = true)